modules/euidIdSystem.js

```diff
@@ -103,7 +103,6 @@ export const euidIdSubmodule = {
       mappedConfig.cstg = {
         serverPublicKey: config?.params?.serverPublicKey,
         subscriptionId: config?.params?.subscriptionId,
-        optoutCheck: 1,
```

Comment:
could you fix these jsdoc warnings too please?

![image](https://github.com/prebid/Prebid.js/assets/1683175/487dc509-588d-4127-91de-d4c6c2671a68)